solver/src/driver.rs

```diff
@@ -80,6 +80,9 @@ impl Driver {
             Some(settlement) => settlement,
         };
         info!("Computed {:?}", settlement);
+        if settlement.trades.is_empty() {
```

Comment:
Should we also check that interactions are empty? I don't think we have defined yet why we would want to settle a trade with just interactions, but I think it's conceptually possible (maybe via some 0x limit orders that are not coming from market makers and would give us positive fees)

Reply:
I don't think we should check it now because right now that is more likely a bug then intentional. If we do have such a use case we can change it.